pkg/apis/eksctl.io/v1alpha5/cloudwatch.go

```diff
@@ -0,0 +1,28 @@
+package v1alpha5
+
+// ClusterCloudWatch contains config parameters related to CloudWatch
+type ClusterCloudWatch struct {
+	//+optional
+	ClusterLogging *ClusterCloudWatchLogging `json:"clusterLogging,omitempty"`
+}
+
+// ClusterCloudWatchLogging container config parameters related to cluster logging
+type ClusterCloudWatchLogging struct {
+	//+optional
+	EnableTypes []string `json:"enableTypes,omitempty"`
```

Comment:
Once we have `--enable-types` and `--disable-types`, we should add `DisableTypes` here.

Reply:
I've opened https://github.com/weaveworks/eksctl/issues/1091.